app/routes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var	express = require('express');
+var	config = require('./controller/settings');
+
+var	router = express.Router();
+
+// Route to UPDATE the config
+router.post('/configs', function (req, res) {
+	var configData = config.update(req);
+	console.log(configData);
+	res.send(configData);
+});
+
+// Route to READ the configs
+router.get('/configs', function (req, res) {
+	var configData = config.data;
```

Comment:
this needs to use config.get you never use config.data outside of the get and set methods!